Cleanup err codes (#67547)
rust-lang/rust: src/librustc_error_codes/error_codes/E0124.md

> A struct with two fields having the same name has been declared.

Reviewer: ```suggestion
A struct was declared with two fields having the same name.
```
